[2024-10-15T16:36:40.121+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-15T16:36:40.139+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres.create_table manual__2024-10-15T16:36:39.777855+00:00 [queued]>
[2024-10-15T16:36:40.152+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres.create_table manual__2024-10-15T16:36:39.777855+00:00 [queued]>
[2024-10-15T16:36:40.153+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-10-15T16:36:40.171+0000] {taskinstance.py:2888} INFO - Executing <Task(PostgresOperator): create_table> on 2024-10-15 16:36:39.777855+00:00
[2024-10-15T16:36:40.179+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres', 'create_table', 'manual__2024-10-15T16:36:39.777855+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres.py', '--cfg-path', '/tmp/tmpvq99y24g']
[2024-10-15T16:36:40.180+0000] {standard_task_runner.py:105} INFO - Job 29: Subtask create_table
[2024-10-15T16:36:40.181+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=2115) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-15T16:36:40.181+0000] {standard_task_runner.py:72} INFO - Started process 2116 to run task
[2024-10-15T16:36:40.231+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_postgres.create_table manual__2024-10-15T16:36:39.777855+00:00 [running]> on host 23605e1f5f41
[2024-10-15T16:36:40.288+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='james' AIRFLOW_CTX_DAG_ID='dag_with_postgres' AIRFLOW_CTX_TASK_ID='create_table' AIRFLOW_CTX_EXECUTION_DATE='2024-10-15T16:36:39.777855+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-15T16:36:39.777855+00:00'
[2024-10-15T16:36:40.288+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-15T16:36:40.296+0000] {sql.py:266} INFO - Executing: 
        create table if not exists dag_runs(
            dt date,
            dag_id character varying,
            primary key (dt, dag_id)
        )
        
[2024-10-15T16:36:40.300+0000] {base.py:84} INFO - Retrieving connection 'postgress_connection'
[2024-10-15T16:36:40.304+0000] {base.py:84} INFO - Retrieving connection 'postgress_connection'
[2024-10-15T16:36:40.307+0000] {sql.py:509} INFO - Running statement: 
        create table if not exists dag_runs(
            dt date,
            dag_id character varying,
            primary key (dt, dag_id)
        )
        , parameters: None
[2024-10-15T16:36:40.318+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-15T16:36:40.319+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres, task_id=create_table, run_id=manual__2024-10-15T16:36:39.777855+00:00, execution_date=20241015T163639, start_date=20241015T163640, end_date=20241015T163640
[2024-10-15T16:36:40.355+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-15T16:36:40.381+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-15T16:36:40.384+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
